native/src/routes/SearchModal.tsx

```diff
@@ -102,7 +111,7 @@ const SearchModal = ({
                   routeType={SEARCH_ROUTE}
                   language={languageCode}
                   cityCode={cityCode}
-                  noResults={searchResults.length === 0}
+                  noResults={limitedResults.length === 0}
                   query={query}
```

Comment:
🤔 Should we use `debouncedQuery` here as well?

Reply:
I think if it makes things slower on web it should be the same here, right? In general I think its good to have consistent behavior anyway.

Reply:
Wait its used already:
https://github.com/digitalfabrik/integreat-app/blob/2323f7bc781cd160bbe8815c0396e4313776e1eb/native/src/routes/SearchModal.tsx#L61-L63

Reply:
yes, but not for the highlight/feedback 🤔